src/llvm/LLVMDG2Dot.h

```diff
@@ -95,6 +95,18 @@ class LLVMDG2Dot : public debug::DG2Dot<LLVMNode>
             os << "\\nERR: no BB";
         }
 
+        //Print Location in source file
+        if (const llvm::Instruction *I = llvm::dyn_cast<llvm::Instruction>(val)) {
+            const llvm::DebugLoc& Loc = I->getDebugLoc();
+            if(Loc) {
+                os << "\" labelURL=\"";
+                std::string stmp;
+                llvm::raw_string_ostream ross(stmp);
+                Loc.print(llvm::cast<llvm::raw_ostream>(ross));
```

Comment:
Here you could simply use
```C
llvm::raw_os_ostream ross(os);
Loc.print(ross);
ross.flush();
```
without creating a new temporary string. Sorry that I didn't catch it before.
Otherwise it looks good.